lib/rules/web/admin_console/4.4/search.xyaml

```diff
@@ -0,0 +1,50 @@
+search_by_resource:
```

Comment:
Kept search_by_resource in search.xyaml, and remove search_by_resource_type and search_by_api_group.